task24/js/main.js

```diff
@@ -0,0 +1,38 @@
+import "../css/style.css";
+
+const modalWrapper = document.getElementById("js-modalContents");
+const modal = document.getElementById("js-modal");
+const modalContents = document.getElementById('js-modalContents');
+const modalPlace = document.getElementById("js-modalPlace");
+const closeModal = document.getElementById("js-modalClose");
+const closeModalText = document.getElementById("js-modalCloseText");
```

Comment:
Q.
Do you use it?

Reply:
Same corrected.
[22ae208](https://github.com/seeeen044/JavaScript_handson/pull/26/commits/22ae208b1021a507ebcac0d073754304b3540df4)